planners:
  - name: GlobalPlanner
    type: global_planner/GlobalPlanner
#  - name: SBPLLatticePlanner
#    type: SBPLLatticePlanner

controllers:
#  - name: DWAPlannerROS
#    type: dwa_local_planner/DWAPlannerROS
  - name: TebLocalPlannerROS
    type: teb_local_planner/TebLocalPlannerROS

controller_frequency: 10.0
controller_patience: 3.0

planner_frequency: 0.3
planner_patience: 3.0
planner_max_retries: 3

recovery_enabled: false

oscillation_timeout: 10.0
oscillation_distance: 0.2


#SBPLLatticePlanner:
#  planner_type: 'ARAPlanner'
#  allocated_time: 10.0 # default: 10.0
#  initial_epsilon: 3.0
#  environment_type: 'XYThetaLattice' # The only environment supported
#  forward_search: true # default is false
#  nominalvel_mpersecs: 0.8
#  timetoturn45degsinplace_secs: 0.6
#  lethal_obstacle: 250
